### PR TITLE
Fixed problems of profiler related with API request handlers

### LINKED ISSUE #71: There is no profiling statement in API handlers
The AirOne implements a simple profiling feature to log following informations for each requests by using the `airone_profile` decorator.

* Who authenticated user access
* Which path is accessed
* How many time it take until a response is returned

But its decorator doesn't apply some API handlers (e.g. AccessTokenAPI). We have to coverage whole request handlers.

### PR DESCRIPTION
**Pull request by userlocalhost**
Close: #71, #72

This commit fixed following previous implementation problems.

* The airone_profile output was degenerated output (no information about
  what type, who sends and which path of is called for calling API handler).
* Some API handlers didn't set airone_profile configuration.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@
 * Fixed the Refer of EntityAttr failing to import
 * Fixed the upper limit when using offset of GET entry API
 * Fixed for API request without token results in exception error
+* Fixed problems of profiler related with API request handlers
```
Comment (hinashi) [inline CHANGELOG.md:13]: Since the master's changelog has been updated, I think it is better to rebase or marge.

**Review by hinashi — COMMENTED** [review]
LGTM!
I confirmed that the API log is output.

**Review by takano32 — APPROVED** [review]
LGTM 👍 

And thanks for your confirming, @hinashi . :)

File: CHANGELOG.md (hunk 1 above)
Comment (userlocalhost) [reply CHANGELOG.md:13]: Thank you for telling my mistake. I update this branch to up-to-date.

**Pull request merged**